README.markdown

```diff
@@ -372,6 +537,31 @@ Constants are defined using the `let` keyword, and variables with the `var` keyw
 
 **Tip:** A good technique is to define everything using `let` and only change it to `var` if the compiler complains!
 
+You can define constants on a type rather than an instance of that type using type properties. To declare a type property as a constant simply use `static let`. Type properties declared in this way are generally preferred over global constants because they are easier to distinguish from instance properties. Example:
+
+**Preferred:**
+```swift
+enum Math {
+  static let e  = 2.718281828459045235360287
+  static let pi = 3.141592653589793238462643
+}
+
+radius * Math.pi * 2 // circumference
```

Comment:
I don't love this example, because in reality you'd just use `.pi`. How about something that's not in the standard library like Avogadro's number?